trin-core/src/jsonrpc.rs

```diff
@@ -219,6 +219,8 @@ fn handle_request(
             "result": "trin 0.0.1-alpha",
         })
         .to_string()),
+        "test_historyNetwork" => dispatch_portal_request(obj, portal_tx),
+        "test_stateNetwork" => dispatch_portal_request(obj, portal_tx),
```

Comment:
is there a cleaner way to combine the "test_historyNetwork", "test_stateNetwork", and the one below this since they all call the same function?

Reply:
Yeah, there is for sure. But, it's not obvious to me currently the best way to clean this up, and I think the optimal refactor will emerge as we add more and more endpoints. I'm partial to leaving as-is for now, and just leaving a comment that refactoring should be handled soon.